app/models/vehicle.rb

```diff
@@ -0,0 +1,6 @@
+class Vehicle < ApplicationRecord
+  belongs_to :user
+
+#I do not think we want to validate presence of storage_rack and awd. I think this would mean that if a user does not have one of these they will be unable to add their car. Maybe? Something to test for.
+  validates_presence_of :make, :model, :year, :total_seats
+end
```

Comment:
Agreed on the comment - I don't think we need to validate for that; they're more optional fields anyways.